src/components/Company-Slider/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useRef, useState } from 'react';
```

Comment:
`No need to import React. Hooks are not being used so again, no need for import`